### ISSUE TITLE
"olk.exe -devtools" is no longer available in the latest version of the new Outlook for Windows.

### ISSUE THREAD

**Issue by t-miyake** [OfficeDev/office-js#4743]
## Environment

* Platform:  PC desktop
* Host :  New Outlook for Windows
* Office version number: 1.2024.725.400
* Operating System: Windows 11
 
## Expected behavior

Previously, "olk.exe -devtools" was available as per the documentation blow.
https://github.com/OfficeDev/office-js/wiki/Collecting-log-files-for-Outlook-Add%E2%80%90ins-issues

## Current behavior

In the latest version I have (1.2024.725.400), "olk.exe -devtools" is no longer available. The error shown in the screenshot below appears.

![image](https://github.com/user-attachments/assets/8f800ed3-3bd3-4992-b10e-2c51b10ea69b)

## Steps to reproduce

1. Open the Run window.
2. Type olk.exe –devtools.
3. Press OK.

## Context

This issue affects my ability to use developer tools in New Outlook. I want to verify whether this is a permanent change or a temporary issue. If it's a specification change, is there an alternative method to use the dev tools?

**Comment by DivyaPatidar** [comment]
@t-miyake Since when are you seeing this behaviour? Can you debug in Outlook Web client meanwhile to unblock yourself?

**Comment by t-miyake** [comment]
@DivyaPatidar I am not sure if it is accurate, but in "1.2024.625.600", the developer tools are displayed as shown in the screenshot. However, in "1.2024.725.400", an error message is displayed as shown in the screenshot.

![OLD](https://github.com/user-attachments/assets/dc2a55d9-c25a-4c3c-af05-985df65cb2cd)

![NEW](https://github.com/user-attachments/assets/451c38c7-576a-40a9-b5f5-387a2240959d)

Of course, many purposes can be achieved by debugging with the Outlook Web client, but if an issue occurs with the New Outlook for Windows, it becomes difficult to investigate.

Moreover, in several responses to issues in this repository, comments from Microsoft members suggest using "olk.exe -devtools" to obtain logs or clear caches for debugging in the New Outlook for Windows.

Furthermore, this is also mentioned in the repository's Wiki (https://github.com/OfficeDev/office-js/wiki/Collecting-log-files-for-Outlook-Add%E2%80%90ins-issues), where it states that log collection for the New Outlook for Windows involves launching the developer tools using "olk.exe -devtools".

If this is an intentional specification change, please update the documentation and inform users of the specification change. If it is a bug, please fix it.

**Comment by t-miyake** [comment]
@DivyaPatidar @exextoc Could you please provide an update on the issue? Thank you in advance for your help.

**Comment by anjalitp** [comment]
@t-miyake  We checked with the concerned team. The above error can occur if a wrong 'dash' character is being used in the command.
olk.exe –devtools ❌
vs
olk.exe -devtools ✅


The first one will cause this error. That character is often auto-corrected by Word and other Office editors if the user types -- which gets converted to — .

Please check if this is causing the issue and let us know.

**Comment by t-miyake** [comment]
@anjalitp  Thank you for checking with the team. I apologize for the confusion; it was a misunderstanding on my part.

Upon review, I found the following:
- olk.exe -devtools ✅
- olk.exe --devtools ✅
- olk.exe –devtools ❌
- olk.exe ---devtools ❌

As you pointed out, the error was due to using an incorrect character. I appreciate your help in identifying this, and I'm sorry for any inconvenience caused. Thank you for your understanding and support.

Given that this resolves the issue, I will close the issue with this comment. Thank you again for your assistance.
